app/experimenter/experiments/tests/test_bugzilla.py

```diff
@@ -44,13 +50,23 @@ def test_creating_pref_bugzilla_ticket_returns_ticket_id(self):
                 "cc": settings.BUGZILLA_CC_LIST,
                 "type": "task",
                 "priority": "P3",
+                "see_also": [
+                    get_bugzilla_id(experiment.data_science_bugzilla_url)
```

Comment:
Try to hardcode as much as possible in your expected output checks, so here hard code the actual ids.